go-get.go

```diff
@@ -0,0 +1,6 @@
+// This file is here so go get succeeds, as without it errors with:
+// no buildable Go source files in ...
+//
+// +build !windows
+
+package wmi
```

Comment:
I'm not sure why this is necessary. I'm ok with those messages, since they aren't errors, just warnings, and go get continues.